README.md

```diff
@@ -1,2 +1,299 @@
-# living-doc-generator
+# Living documentation generator
+
+- [Motivation](#motivation)
+- [Usage](#usage)
+    - [Prerequisites](#prerequisites)
+    - [Adding the Action to Your Workflow](#adding-the-action-to-your-workflow)
+- [Action Configuration](#action-configuration)
+    - [Environment Variables](#environment-variables)
+    - [Inputs](#inputs)
+    - [Features de/activation](#features-deactivation)
+    - [Features configuration](#features-configuration)
+    - [Page generator options](#page-generator-options)
+- [Action Outputs](#action-outputs)
+- [Project Setup](#project-setup)
+- [Run unit test](#run-unit-test)
+- [Deployment](#deployment)
+- [Features](#features)
+    - [Data Mining from GitHub Repositories](#data-mining-from-github-repositories)
+    - [Data Mining from GitHub Projects](#data-mining-from-github-projects)
+    - [Living Documentation Page Generation](#living-documentation-page-generation)
+- [Contribution Guidelines](#contribution-guidelines)
+- [License Information](#license-information)
+- [Contact or Support Information](#contact-or-support-information)
+
 A tool designed to data-mine GitHub repositories for issues containing project documentation (e.g. tagged with feature-related labels). This tool automatically generates comprehensive living documentation in markdown format, providing detailed feature overview pages and in-depth feature descriptions.
+
+## Motivation
+Addresses the need for continuously updated documentation accessible to all team members and stakeholders. Achieves this by extracting information directly from GitHub issues and integrating this functionality to deliver real-time, markdown-formatted output. Ensures everyone has the most current project details, fostering better communication and efficiency throughout development.
+
+## Usage
+### Prerequisites
+Before we begin, ensure you have a GitHub Token with permission to fetch repository data such as Issues and Pull Requests.
+
+### Adding the Action to Your Workflow
+
+See the default action step definition:
+
+```yaml
+- name: Generate Living Documentation
```

Comment:
May change name to `Generate Living Documentation` everywhere in README as it is in action.yml file.

Reply:
I have checked the file and fixed capital in 1st line. Other variations in the file still make sense to me.
Please provide reasons for the change in local meanings.